source/CardGames.py

```diff
@@ -161,3 +175,34 @@ def dealCards(self, numCards: int, players: "list[Player]"):
   def resetDeck(self):
     self.deck.reset()
     self.deck.shuffle()
+
+# simple respond for dealer in single player with known hands 
+def dealerHand(dealer, house, player):
+  houseHand = house.handSum()
+  playerHand = player.handSum()
+  if playerHand == 21 and len(player.hand) == 2: #if blackjack then player auto win
+    return 1   
+  if houseHand == 21 and len(house.hand) == 2: #if blackjack then house auto win
+    return -1 
+  while houseHand < 16: #force hit when below 16
```

Comment:
The house should hit while houseHand < 17

Reply:
fixed